fmriprep/workflows/bold/base.py

```diff
@@ -295,7 +285,11 @@ def init_func_preproc_wf(bold_file):
     # Generate a tentative boldref
     bold_reference_wf = init_bold_reference_wf(omp_nthreads=omp_nthreads)
     bold_reference_wf.inputs.inputnode.dummy_scans = config.workflow.dummy_scans
-    if sbref_file is not None:
+    if sbref_files is not None:
+        from niworkflows.interfaces.images import ValidateImage
+        val_sbref = pe.MapNode(ValidateImage(), name='val_sbref',
+                               iterfield=['in_file'])
+        val_sbref.inputs.in_file = sbref_files
```

Comment:
shouldn't this be done within the `bold_reference_wf` (as it is done for the bold datasets)?

Reply:
(if @tsalo agrees on this, I'll take care of it)

Reply:
That sounds good. Thank you!

Reply:
just pushed a commit to nipreps/niworkflows#408